setup/stock_quant_package_dimension_total_weight_from_packaging/odoo/addons/stock_quant_package_dimension_total_weight_from_packaging

```diff
@@ -0,0 +1 @@
+../../../../stock_quant_package_dimension_total_weight_from_packaging
```

Comment:
For the next migration, these changes should go into a separate commit.

See https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-16.0#technical-method-to-migrate-a-module-from-150-to-160-branch

Reply:
Thank @rousseldenis , i will take notes for the next time

Reply:
Thank @rousseldenis , I updated